src/app/core/session/auth/keycloak/keycloak-auth.service.ts

```diff
@@ -144,6 +144,33 @@ export class KeycloakAuthService extends AuthService {
       }
     );
   }
+
+  createUser(username: string, email: string, roles: Role[] = []) {
+    return this.httpClient.post(`${environment.account_url}/account`, {
+      username,
+      email,
+      roles,
+    });
+  }
+
+  updateUser(userId: string, user: any) {
+    return this.httpClient.put(
+      `${environment.account_url}/account/${userId}`,
+      user
+    );
+  }
+
+  getUser(username: string) {
```

Comment:
```suggestion
  getUser(username: string): Observable<{ id: string; email: string; roles: Role[] }> {
```

or, alternatively, create an interface describing the returned object